.mergify.yml

```diff
@@ -65,6 +77,7 @@ pull_request_rules:
     conditions:
       - base=master
       - label=automerge:rebase
+      - linear-history
     actions:
       queue:
         merge_method: merge
```

Comment:
I know this is the default, but should we specify `autosquash: true` here too to make things more obvious?

Reply:
that field is deprecated and will probably wont be supported in a few months time like the other deprecated fields